src/Microsoft.Health.Dicom.Api/Features/Filters/BodyModelStateValidatorAttribute.cs

```diff
@@ -17,7 +17,7 @@ public override void OnActionExecuting(ActionExecutingContext context)
             EnsureArg.IsNotNull(context, nameof(context));
             if (!context.ModelState.IsValid)
             {
-                throw new InvalidRequestBodyException(string.Join(",", context.ModelState.SelectMany(x => x.Value.Errors).Select(x => x.ErrorMessage)));
+                throw new InvalidRequestBodyException(string.Join("/n", context.ModelState.SelectMany(x => x.Value.Errors).Select(x => x.ErrorMessage)));
```

Comment:
* Environment.NewLine instead of "/n"?  btw, seems "/n" doesn't work 
* Add a new line just after _Details:_, so that all error message are in individual lines

![image](https://user-images.githubusercontent.com/67345894/115934968-0ba61880-a447-11eb-8a89-910f232e446e.png)